### ISSUE TITLE
Mdapt/Gdapt shaders

### ISSUE THREAD

**Issue by Defreezer** [akgunter/crt-royale-reshade#31]
Hello. I hate to have to ask it here, but would you consider porting sp00kyfox's Mdapt and Gdapt shaders to Reshade later as well, if/when you have the time?
https://forums.libretro.com/t/mdapt-gdapt-dithering-treatment-updated-06-06-14/375
They are a crucial addition to CRT shaders including Royale, but the guy keeping the RSRetroarch repo is no longer working on it, it seems.
Good work on the Royale port so far, by the way.

**Comment by akgunter** [comment]
It's funny that you mention that. I just finished MDAPT on Friday and sent Matsilagi a pull request yesterday.

I think he's working on GDAPT? If not, I'll be able to do that one later. It'll be easier than MDAPT.

In the meantime, you can grab MDAPT from my RSRetroArch fork: https://github.com/akgunter/RSRetroArch/blob/main/Shaders/mdapt.fx. Remember to set the scale correctly, and feel free to post any issues over in that repo.

And I'm glad to hear you like the port!

**Comment by akgunter** [comment]
Nevermind, he just merged my PR. Grab it from his repo, since that's now the "official" version.

**Comment by Defreezer** [comment]
Very nice, thanks.

**Comment by Defreezer** [comment]
Hey, uh. Sorry to bother you, I just got around to testing the shader on Duckstation (D3D11) with Reshade 5.6 and it doesn't seem to be taking any effect? None of the settings seem to change anything, not sure if I'm doing something wrong. I couldn't open an issue on your other repository and RSRetroarch's issue tracker is a ghost town so I had to do it here, hope you don't mind. Is Gdapt still on the cards?
